Formula/density.rb

```diff
@@ -0,0 +1,30 @@
+class Density < Formula
+  desc "Superfast compression library"
+  homepage "https://groups.google.com/d/forum/centaurean-density"
+  url "https://github.com/centaurean/density/archive/density-0.14.2.tar.gz"
+  sha256 "0b130ea9da3ddaad78810a621a758b47c4135d91d5b5fd22d60bbaf04bc147da"
+
+  resource "cputime" do
+    url "https://github.com/centaurean/cputime.git",
+      :revision => "d435d91b872a4824fb507a02d0d1814ed3e791b0"
+  end
+
+  resource "spookyhash" do
+    url "https://github.com/centaurean/spookyhash.git",
+      :revision => "2c27b434d097198cab2272d4160b413a5f139049"
+  end
+
+  def install
+    (buildpath/"benchmark/libs/cputime").install resource("cputime")
+    (buildpath/"benchmark/libs/spookyhash").install resource("spookyhash")
+    system "make"
+    include.install "src/density_api.h"
+    share.install "build/benchmark"
```

Comment:
Unless `benchmark` is a really integral part of this package, I'm :+1: on not installing this (and just using the build path to run it in the `test` block),

Reply:
Isn't the build path subject to deletion on `brew cleanup`, etc.? I only installed `benchmark` because there's no good way to get it later short of downloading the entire package and building it on-the-fly.

Reply:
Yeah, I forgot we don't allow access to `buildpath` outside of `install`. This is fine, then.